test/e2e/templates/scala/radio_scalabuilddc.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Define a bunch of functions and set a bunch of variables
+TEST_DIR=$(readlink -f `dirname "${BASH_SOURCE[0]}"` | grep -o '.*/oshinko-s2i/test/e2e')
+source $TEST_DIR/common
+
+SCRIPT_DIR=$(readlink -f `dirname "${BASH_SOURCE[0]}"`)
+source $SCRIPT_DIR/../builddc
+
+RESOURCE_DIR=$TEST_DIR/resources
+set_template $RESOURCE_DIR/oshinko-scala-spark-build-dc.yaml
+set_git_uri https://github.com/pdmack/scala-sbt-s2i-test
+set_worker_count $S2I_TEST_WORKERS
+set_fixed_app_name scala-build
+set_app_main_class org.apache.spark.examples.SparkPi
+
+# Clear these
+set_app_file
+set_exit_flag
+
+# Need a little preamble here to read the resources.yaml, create the pyspark template, and save
+# it to the resources directory
+set +e
+oc create -f https://radanalytics.io/resources.yaml
+oc export template oshinko-scala-spark-build-dc > $RESOURCE_DIR/oshinko-scala-spark-build-dc.yaml
+set -e
+
+os::test::junit::declare_suite_start "$MY_SCRIPT"
+
+# Verify that pod goes to "Completed" if app_exit is set
+echo "++ test_exit"
+test_fixed_exit
+
+echo "++ test_cluster_name"
+test_cluster_name
+
+echo "++ test_del_cluster"
+test_del_cluster
+
+# Tests the ability to set arbitrary app arg strings
+echo "++ test_app_args"
+test_app_args
+
+echo "++ test_pod_info"
+test_podinfo
+
+echo "++ test_named_config"
+test_named_config
+
+echo "++ test_driver_config"
+test_driver_config
+
+echo "++ test_spark_options"
+test_spark_options
+
+echo "++ test_no_app_name"
+#test_no_app_name
```

Comment:
Add an issue here to make the scala resources.yaml match pyspark/java and reference it here, the test can be added back in when the issue is fixed